.github/workflows/detect-breaking-changes-build.yml

```diff
@@ -11,37 +11,40 @@ jobs:
         working-directory: './pr'
```

Comment:
I applied prettier to this yml file. Please use "hide white spaces to see the changes better"